spec/fixtures/facebook_video_oembed.json

```diff
@@ -5,10 +5,9 @@
   "provider_name": "Facebook",
   "success": true,
   "height": 281,
-  "html": "\u003Cdiv id=\"fb-root\">\u003C/div>\n\u003Cscript>(function(d, s, id) {\n  var js, fjs = d.getElementsByTagName(s)[0];\n  if (d.getElementById(id)) return;\n  js = d.createElement(s); js.id = id;\n  js.src = \"//connect.facebook.net/es_ES/sdk.js#xfbml=1&version=v2.10\";\n  fjs.parentNode.insertBefore(js, fjs);\n}(document, 'script', 'facebook-jssdk'));\u003C/script>\u003Cdiv class=\"fb-video\" data-href=\"https://www.facebook.com/facebook/videos/1814600831891266\">\u003Cblockquote cite=\"https://www.facebook.com/Devex/videos/1814600831891266/\" class=\"fb-xfbml-parse-ignore\">\u003Ca href=\"https://www.facebook.com/Devex/videos/1814600831891266/\">Power with Purpose 2017\u003C/a>\u003Cp>At Devex, we&#039;re committed to celebrating women leaders in our industry. That&#039;s why we&#039;ve once again partnered with McKinsey &amp; Company to tell the stories of five remarkable women through Power with Purpose, our leadership recognition campaign.\n\nHelp us celebrate women leaders in global development, and share #PowerWithPurpose: http://powerwithpurpose.devex.com\u003C/p>Publicado por \u003Ca href=\"https://www.facebook.com/Devex/\">Devex\u003C/a> en Jueves, 22 de junio de 2017\u003C/blockquote>\u003C/div>",
+  "html": "<div id=\"fb-root\"></div>\n<script>(function(d, s, id) {\n  var js, fjs = d.getElementsByTagName(s)[0];\n  if (d.getElementById(id)) return;\n  js = d.createElement(s); js.id = id;\n  js.src = \"//connect.facebook.net/es_ES/sdk.js#xfbml=1&version=v2.10\";\n  fjs.parentNode.insertBefore(js, fjs);\n}(document, 'script', 'facebook-jssdk'));</script><div class=\"fb-video\" data-href=\"facebook.com/facebook/videos/1814600831891266\"><blockquote cite=\"https://www.facebook.com/Devex/videos/1814600831891266/\" class=\"fb-xfbml-parse-ignore\"><a href=\"https://www.facebook.com/Devex/videos/1814600831891266/\">Power with Purpose 2017</a><p>At Devex, we&#039;re committed to celebrating women leaders in our industry. That&#039;s why we&#039;ve once again partnered with McKinsey &amp; Company to tell the stories of five remarkable women through Power with Purpose, our leadership recognition campaign.\n\nHelp us celebrate women leaders in global development, and share #PowerWithPurpose: http://powerwithpurpose.devex.com</p>Publicado por <a href=\"https://www.facebook.com/Devex/\">Devex</a> en Jueves, 22 de junio de 2017</blockquote></div>",
```

Comment:
I would just truncate that.

Reply:
It needs to include the full thing, otherwise the integration test would fail (the exported reference document includes them fully).

Reply:
Well, lets hope those tests never break and someone has to figure out the difference.